.github/renovate.json

```diff
@@ -11,9 +11,23 @@
         "k8s.io/api"
       ],
       "updateTypes": [
-        "major"
+        "major",
+        "minor",
+        "digest"
       ],
       "enabled": false
     }
+    {
+      "packageNames": [
+        "k8s.io/apimachinery",
+        "k8s.io/client-go",
+        "k8s.io/api",
+        "k8s.io/utils"
```

Comment:
Not sure I understand the config options fully, but why it `k8s.io/utils` not in the disabled list above?

Reply:
I messed up. It doesn't need to be part of that list because it is versioned independently (or rather, not at all), but it did need to have its digest patches disabled. If they ever start releasing versioned updates then we will get them